game/src/plugins/skills/mining/src/mining.plugin.kts

```diff
@@ -50,23 +60,33 @@ class MiningAction(val player: Player, val p: Position, val ore: Ore) : Distance
         }
     }
 
-    fun findPickaxe(): Pickaxe? { // Find the best pick the player has
+    private fun findPickaxe(): Pickaxe? {
         for (id in getPickaxes()) {
+            val pick = lookupPickaxe(id) //Will not return null
```

Comment:
How do we know this won't return `null`?

Reply:
Because getPickaxes returns the set of ID values that lookupPickaxe looks too.